File: fuckingRetrofit/app/src/main/java/com/example/fuckingretrofit/room/AppDatabase.java
package com.example.fuckingretrofit.room;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import com.example.fuckingretrofit.pojo.UserData;

@Database(entities = {UserData.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract UserDao userDao();
}

File: AndroidAvito/app/src/main/java/com/example/meteohubapp/data/db/AppDatabase.kt
package com.example.meteohubapp.data.db

import androidx.room.Database
import androidx.room.RoomDatabase
import com.example.meteohubapp.domain.our_model.City

/**
 * @author Bulat Bagaviev
 * @created 22.10.2022
 */
@Database(entities = [City::class], version = 1)
abstract class AppDatabase: RoomDatabase() {
    abstract fun cityDao(): CityDao
}